samples/suit/smp_transfer/README.rst

```diff
@@ -22,20 +22,20 @@ The sample supports the following development kit:
 You need the nRF Device Manager app for SUIT update over Bluetooth Low Energy:
 
 * `nRF Device Manager mobile app for Android`_
-  (The minimum required version is v2.0.)
+  (The minimum required version is v2.0.0.)
 
 * `nRF Device Manager mobile app for iOS`_
-  (The minimum required version is v1.7.)
+  (The minimum required version is v1.7.0.)
 
-For a SUIT update over UART, you need to install :ref:`zephyr:mcu_mgr`, a tool that can be used to upload SUIT envelopes through the SMP protocol.
+For a SUIT update over UART through the SMP protocol, you need to install SUIT commands to nrfutil by ``nrfutil install suit`` command.
```

Comment:
Should we also add some minimal version of nrfutil suit?

(v2.8.0-rc1.1) C:\ncs_new\v2.8.0-rc1.1> nrfutil suit --version
nrfutil-suit 0.8.1 (6433d59 2024-10-11)

Reply:
Yes, will add it.